resilience4j-ratelimiter/src/main/java/io/github/resilience4j/ratelimiter/internal/SemaphoreBasedRateLimiter.java

```diff
@@ -260,6 +262,18 @@ private void publishRateLimiterAcquisitionEvent(boolean permissionAcquired, int
         eventProcessor.consumeEvent(new RateLimiterOnFailureEvent(name, permits));
     }
 
+    /**
```

Comment:
This one looks very good, I'm ready to merge it, but can you please add a mention about this method to this class doc at the top of a file.